ai-platform-unified/notebooks/official/custom-training-batch-prediction.ipynb

```diff
@@ -0,0 +1,1213 @@
+{
```

Comment:
<p>Please update the "Objective" section to fit the batch prediction workflow - since this tutorial doesn't include the model deployment process that's required for online prediction</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/281/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>Done. I should fix the online notebook as well. Good catch.</p>